src/components/Layout/Layout.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Layout from './Layout.component';
+import Theme from '../App/App.styled';
+
+describe('Layout Component Tests', () => {
+  const setup = () => {
+    const utils = render(
+      <Theme>
+        <Layout />
+      </Theme>
+    );
+    return {
+      ...utils,
+    };
+  };
```

Comment:
Yes :D I reconsider what I was doing and I better changed to use `screen` because you don't need to deconstruct to get the utils and use the beforeEach jest function to run the render before each unit test